### PR TITLE
aws-backup: add German translation

### PR DESCRIPTION
**Pull request by Slotharmy**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Slotharmy** [comment]
Hello, can some pls make the approving reviews?

File: pages.de/common/aw-backup.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# aws backup
+
+> Einheitliches Backup-Service zum Schutz der Amazon Web Services und der damit verbundenen Daten.
+> Weitere Informationen: <https://docs.aws.amazon.com/cli/latest/reference/backup/index.html>.
+
+- Rückgabe von Backup-Plan-Details für eine bestimmte Backup-Plan-ID:
```
Comment (pixelcmtd) [inline pages.de/common/aw-backup.md:6]:
```suggestion
- Gib Backup-Plan-Details für eine bestimmte Backup-Plan-ID aus:
```

File: pages.de/common/aw-backup.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`aws backup get-backup-plan --backup-plan-id {{id}}`
+
+- Erstellen eines Sicherungsplans unter Verwendung eines bestimmten Sicherungsplannamens und von Sicherungsregeln:
```
Comment (pixelcmtd) [inline pages.de/common/aw-backup.md:10]:
```suggestion
- Erstelle einen Backup-Plan unter Verwendung eines bestimmten Backup-Plan-Namens und von Backup-Regeln:
```

File: pages.de/common/aw-backup.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`aws backup create-backup-plan --backup-plan {{plan}}`
+
+- Löschen eines bestimmten Backup-Plans
```
Comment (pixelcmtd) [inline pages.de/common/aw-backup.md:14]:
```suggestion
- Lösche einen bestimmten Backup-Plan:
```

File: pages.de/common/aw-backup.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`aws backup delete-backup-plan --backup-plan-id {{id}}`
+
+- Liefert eine Liste aller aktiven Backup-Pläne für das aktuelle Konto:
```
Comment (pixelcmtd) [inline pages.de/common/aw-backup.md:18]:
```suggestion
- Gib eine Liste aller aktiven Backup-Pläne für das aktuelle Konto aus:
```

File: pages.de/common/aw-backup.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`aws backup list-backup-plans`
+
+- Zeigt Deatuls über die Reportaufträge an:
```
Comment (pixelcmtd) [inline pages.de/common/aw-backup.md:22]:
```suggestion
- Zeige Details über die Report-Aufträge an:
```

**Review by pixelcmtd — CHANGES_REQUESTED** [review]

**Review by pixelcmtd — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**